nursery/get-tcp-endpoints.yml

```diff
@@ -0,0 +1,10 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: get TCP endpoints
+    namespace: collection/network
+    author: michael.hunhoff@fireeye.com
+    scope: basic block
```

Comment:
```suggestion
    scope: basic block
    att&ck:
      - Discovery::Network Service Scanning [T1046]
```

Reply:
not technically "scanning" though it fits the description

>  "Adversaries may attempt to get a listing of services running on remote hosts"

Reply:
hmm `GetExtendedTcpTable` only lists the local system's existing TCP connections and listeners which to me doesn't fit well under "listing of services running on remote hosts". Thoughts?

Reply:
ok, thats fine by me